openwisp_monitoring/check/migrations/0004_create_config_applied.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+from openwisp_monitoring.check.settings import AUTO_CONFIG_CHECK
+from openwisp_monitoring.check.tasks import auto_create_config_check
+
+
+def add_config_applied_checks(apps, schema_editor):
+    if not AUTO_CONFIG_CHECK:
+        return
+    Device = apps.get_model('config', 'Device')
+    for device in Device.objects.all():
+        auto_create_config_check.delay(
+            model=Device.__name__.lower(),
+            app_label=Device._meta.app_label,
+            object_id=str(device.pk),
+        )
+
+
+def remove_config_applied_checks(apps, schema_editor):
+    Check = apps.get_model('config', 'Device')
```

Comment:
**TOTALLY BOGUS**. 19 days should have been enough to test this and make it work but yet, you fail at this Hardik. :-1: you're still sending broken code for final review after all the discussions we had. You have to stop doing this.

Reply:
1. try to run this
2. make sure it deletes also the Metric (and related alert setting cascades)